library/test/src/cli.rs

```diff
@@ -30,7 +31,7 @@ pub struct TestOpts {
 impl TestOpts {
     pub fn use_color(&self) -> bool {
         match self.color {
-            ColorConfig::AutoColor => !self.nocapture && isatty::stdout_isatty(),
+            ColorConfig::AutoColor => !self.nocapture && Stdout::is_atty(),
```

Comment:
As written this will break all platforms for which there is no `IsAtty` implementation yet.